projects/project_extra.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
@TMCamp Is this what you were talking about needing additional input from the user?

Reply:
But isn't that effectively what this is doing? Was there an issue in allowing for user input, or did you want to save that for a future iteration or something?

Reply:
@TMCamp Why mark this as resolved? The question wasn't answered.

Reply:
Asking the user to input the roles that they have listed in CFACTS is not really acceptable because we are asking the user to maintain data in separate systems and increasing the risk for errors. Until we can import the Project data in toto this should work because it adds roles that are common across most CMS projects.

Reply:
Ok, I think I understand that, but why the JSON file? It's so small that it seems like it would be far more useful to define as a dictionary in python rather than reading it in on each of these requests. 

Even further, why parse this structure at all if they're meant as defaults (effectively)? Is the parsing meant to be something that would also parse future user-supplied data?

Reply:
Yes, I thought that it would be easier to parse future user-supplied data and/or one could just replace the file to add/change defaults rather than updating hardcoded values.